codeflash/verification/parse_test_output.py

```diff
@@ -40,6 +40,30 @@ def parse_func(file_path: Path) -> XMLParser:
 matches_re_end = re.compile(r"!######(.*?):(.*?)([^\.:]*?):(.*?):(.*?):(.*?)######!")
 
 
+start_pattern = re.compile(r"!\$######([^:]*):([^:]*):([^:]*):([^:]*):([^:]+)######\$!")
```

Comment:
why is this pattern different from the one above? i expect the regexes to be the same